packages/blade/src/components/Menu/VisualSubComponents/MenuDivider.web.tsx

```diff
@@ -0,0 +1,8 @@
+import type { StyledPropsBlade } from '~components/Box/styledProps';
+import { Divider } from '~components/Divider';
+
+const MenuDivider = (styledProps: StyledPropsBlade): React.ReactElement => {
+  return <Divider marginLeft="-8px" marginY="spacing.1" marginRight="-8px" {...styledProps} />;
```

Comment:
use spacing tokens and add a negation before it?

Reply:
done